guides/animate.md

```diff
@@ -0,0 +1,22 @@
+# Alternatives to `$.animate()`
+
+
+## Option 1:
+
+Use CSS animations instead toggled with JS classList#.
```

Comment:
Seems to be missing the last part: `Use CSS animations instead toggled with JS classList#add and classList#remove.`